src/Routes.tsx

```diff
@@ -19,7 +20,8 @@ interface Path {
 export const linkTo = {
     integrations: () => '/integrations',
     notifications: (bundle: string) => `/notifications/${bundle}`,
-    eventLog: (bundle?: string) => `/notifications/eventlog${bundle ? `?bundle=${bundle}` : ''}`
+    eventLog: (bundle?: string) => `/notifications/eventlog${bundle ? `?bundle=${bundle}` : ''}`,
+    splunk: () => '/integrations/splunkSetup'
```

Comment:
could you use `splunk-setup` for the url? :D

Reply:
Done!